app/services/transaction.service.js

```diff
@@ -52,6 +53,8 @@ const logCsvFileStreamComplete = function logCsvFileStreamComplete (timestampStr
     method: 'future',
     gateway_account_ids: gatewayAccountIds,
     multiple_accounts: gatewayAccountIds.length > 1,
+    internal_user: user.internalUser,
+    all_live_service_transactions: allLiveServiceTransactions,
```

Comment:
can we log the number of live services for the user here too?

Reply:
Updated